examples/lighting-app/nrfconnect/main/AppTask.cpp

```diff
@@ -344,6 +352,22 @@ void AppTask::StartThreadHandler(AppEvent * aEvent)
 #endif
 }
 
+void AppTask::StartBLEAdvertisementHandler(AppEvent * aEvent)
+{
+    if (aEvent->ButtonEvent.PinNo != BLE_ADVERTISEMENT_START_BUTTON)
+        return;
+
+    if (!chip::DeviceLayer::ConnectivityMgr().IsBLEAdvertisingEnabled())
+    {
+        chip::DeviceLayer::ConnectivityMgr().SetBLEAdvertisingEnabled(ConnectivityManager::kCHIPoBLEServiceMode_Enabled);
```

Comment:
There's using namespace ::chip::DeviceLayer; at the top so you should be able to refer to ConnectivityMgr() directly.

Reply:
Done